### ISSUE TITLE
nw.cc: last lines not triggered by a test

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/swarm#142]
I don't think I can make up a toy-example that could trigger that error in `nw.cc`:
```
       53:  285:  if (score != dist)
        -:  286:  {
    #####:  287:    fprintf(stderr, "WARNING: Error with query no %" PRIu64 " and db sequence no %" PRIu64 ":\n", queryno, dbseqno);
    #####:  288:    fprintf(stderr, "Initial and recomputed alignment score disagreement: %" PRId64 " %" PRId64 "\n", dist, score);
    #####:  289:    fprintf(stderr, "Alignment: %s\n", cigar);
        -:  290:  }

```
Am I correct? or do you happen to have such an example?

**Comment by torognes** [comment]
In this case it should be an assert. This error should not occur unless there is a bug in the code.

I'll change it to an assert.

**Comment by torognes** [comment]
Fixed in commit 9286af7c120f9f319dd4f7eff9072e2b7c15a79f.

**Comment by torognes** [comment]
Sorry, too quick there. Fixed in commit ece391365b01f395e76db7a6b4425a8d4b2d48b5.

**Comment by frederic-mahe** [comment]
```
File 'nw.cc'
Lines executed:100.00% of 121
```
Nice!
